StuffNZ/stuff-salesforce
# Demo purposes

**User Story**

**Issue on Jira**: https://test.com/SSU-131

## Summary of changes
- Pull request is created for demo purposes

## How to Test
- Create an account record
- Click the button Send to Genera
- Click Send and should be successful
